src/app/pages/pages.component.ts

```diff
@@ -14,4 +53,22 @@ import { MENU_ITEMS } from './pages-menu';
 export class PagesComponent {
 
   menu = MENU_ITEMS;
+
+  // TODO: Loop through Menu and translate each one
```

Comment:
You might want to have a look at [this gist](https://gist.github.com/dizco/0182af54fa90e9b1e4d7751c60a2a787)

Reply:
We need to do this according to best practice

Reply:
@tomavic let's do it as part of this PR, what do you think?